README.md

```diff
@@ -1,2 +1,38 @@
+<p align="center">
+  <img width="626px" height="280px" src="assets/logo.png">
+</p>
+
 # Pulsarc
-A community-created rhythm game! Development Discord: https://discord.gg/SYfpvfJ
+[![Discord](https://discordapp.com/api/guilds/486933399425122318/widget.png?style=shield)](https://discord.gg/SYfpvfJ)
+>A community-created rhythm game!
+
+## About
+[Read more about this project!](docs/ABOUT.md)
+
+## How to Play the Prototype
+When the prototype is finished we will have a public release build people can use.
+If you want to play now though, you need:
+
+**Visual Studio Community** - https://docs.microsoft.com/en-us/visualstudio/releasenotes/vs2017-relnotes
+
+**.NET Core 2.2 SDK (v2.2.108)** - https://dotnet.microsoft.com/download/thank-you/dotnet-sdk-2.2.108-windows-x64-installer
+
+**MonoGame 3.7.1 for VisualStudio** - http://community.monogame.net/t/monogame-3-7-1-release/11173
+
+When you have those downloaded and installed, go to the Prototype-branch here: https://github.com/PulsarcGame/Pulsarc/tree/MonoGame-Prototype-Adri. Click the Green "Clone or download" button, then click "Download ZIP" on the bottom right. Extract the .zip into a folder. Open Pulsarc.sln using Visual Studio. When it's done loading click on the little green play icon near the top.
+
+Use the scroll wheel to move the song select up or down. Click on the map you want to play to play it.
+Hit arcs using D - Left, F - Up, J - Down, K - Right. If you want to change the keys, you can edit them starting at line 121 of GameplayEngine.cs, which is found in Pulsarc\UI\Screens\Gameplay.
```

Comment:
This note is temporary and will be likely changed in later versions of the game, I don't think this should be in README.
Instead of telling users to directly edit code, perhaps we should wait until we have a proper configuration system in place, and have a wiki on how to use it.

Reply:
Yeah I was iffy writing that. I don't like telling users to code so they can play the prototype either. I'll remove it when I get on my PC.